packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/FLTCam.h

```diff
@@ -99,7 +99,8 @@ NS_ASSUME_NONNULL_BEGIN
        withCompletion:(void (^)(FlutterError *_Nullable))completion
     NS_SWIFT_NAME(setFocusPoint(_:completion:));
 - (void)setExposureOffset:(double)offset;
-- (void)startImageStreamWithMessenger:(NSObject<FlutterBinaryMessenger> *)messenger;
+- (void)startImageStreamWithMessenger:(NSObject<FlutterBinaryMessenger> *)messenger
+                       withCompletion:(nonnull void (^)(FlutterError *_Nullable))completion;
```

Comment:
Objective-C naming does not repeat "with" in signatures, so this should be `completion:`, not `withCompletion:`.